specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -365,7 +365,6 @@
         "confidenceScore": {
           "type": "number",
           "format": "double",
-          "x-ms-client-name": "confidence",
```

Comment:
You should keep this. Our API archboard doesn't want "score" in the name. This was intentional.

Reply:
I added the missing `x-ms-client-name` property wherever we have confidenceScore.